ansible/roles/ocp4-workload-summit-api-cicd/templates/global-operators.j2

```diff
@@ -0,0 +1,8 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  annotations:
+    olm.providedAPIs: Build.v1.camel.apache.org,CamelCatalog.v1.camel.apache.org,Config.v1alpha1.operator.tekton.dev,Integration.v1.camel.apache.org,IntegrationKit.v1.camel.apache.org,IntegrationPlatform.v1.camel.apache.org
+  name: global-operators
+  namespace: openshift-operators
```

Comment:
Hm. Will this work if another global-operators operator group is already there?

Reply:
Yes, this assumes no other operators exist. Is taking an approach of using CRD and oc create a better approach?

Reply:
I think as long as you're the only workload on that cluster... it may be problematic on a shared cluster (like RHPDS)